ptf/platform_helper/brcm_t0_sai_helper.py

```diff
@@ -262,6 +265,77 @@ def create_host_intf(self):
                 print("Cannot create hostif, error : {}".format(e))
 
 
+    def config_vlan(self):
+        """
+        Default configuation, with Vlan.
+        Craete bridge_port and route interface
+        Steps:
+        """
+        VLAN_ID = 1000
+        PC_PORT_MTU=9100
+        vlan_id = sai_thrift_create_vlan(self.client, vlan_id=VLAN_ID)
+        for i in range(3, 24):
+            bridge_port_id = sai_thrift_create_bridge_port(self.client,
```

Comment:
for each kind of object, please create them in one method as a group, and create those groups in a sequence.
like
1.1 create port
1.2 create vlan
2.1 set hostif
2.2 set port attribute